commerce_spo.info.yml

```diff
@@ -0,0 +1,9 @@
+name: Commerce Single Page Order
+type: module
+description: 'Provides the ability to create and checkout an order within a single page.'
+package: Commerce (contrib)
+core: 8.x
+dependencies:
+  - commerce:commerce_order
+  - commerce:commerce_price
```

Comment:
**CHANGE REQUEST**

`commerce_order` has `commerce_price` as a dependency, no need to add it explicitly. We'll probably need to add other commerce modules but that can be done as we add feature. Just keep `commerce_order` here.